lib/dashlib.py

```diff
@@ -105,6 +105,15 @@ def create_superblock(proposals, event_block_height, budget_max, sb_epoch_time):
     fudge = SUPERBLOCK_FUDGE_WINDOW  # fudge-factor to allow for slighly incorrect estimates
 
     payments = []
+
+    sb_empty = Superblock(
+        event_block_height=event_block_height,
+        payment_addresses='',
+        payment_amounts='',
+        proposal_hashes='',
+    )
+    data_length = len(sb_empty.dashd_serialise()) - 6
```

Comment:
Where does the '6' come from here?

Reply:
`6` is number of bytes we shouldn't count because '|' separator is not used before first address, amount and hash. Probably I should add comment for this line.